cmd/bpf2go/main.go

```diff
@@ -297,6 +302,21 @@ func (b2g *bpf2go) convertAll() (err error) {
 		}
 	}
 
+	if b2g.gopkgIncludes {
+		mods, err := listMods(b2g.outputDir, "all")
+		if err != nil {
+			return fmt.Errorf("listing go modules: %w", err)
+		}
+		vfs, err := createVfs(mods)
+		if err != nil {
+			return fmt.Errorf("creating LLVM vfsoverlay: %w", err)
+		}
+		b2g.vfsoverlay, err = persistVfs(b2g.outputDir, vfs)
```

Comment:
It is actually stored in user cache dir under sha256 of the output path as the name. Changing slightly so that the code doesn't suggest that something is being written to `outputDir`.
```go
vfsID := sha256.Sum256([]byte(b2g.outputDir))
b2g.vfsoverlay, err = persistVfs(vfsID, vfs)
```
(This is being done to reduce the number of files that linger in user cache dir. For the rationale to keep these files around see sibling thread.)